apps/web/src/components/TweetInfo.tsx

```diff
@@ -65,13 +79,58 @@ const TweetInfo = ({ tweet }: TweetInfoProps) => {
                 {`@${tweet.userInfo.username}`}
               </Text>
             </Box>
-          </Flex>
-          <Text mt={2}>{decodeHTML(tweet.text)}</Text>
-          <Text color="gray.500" marginTop={2}>
-            {tweetTimeInfo()}
-          </Text>
+          </Link>
         </Flex>
-      </Link>
+        <Box zIndex={1} pointerEvents="none">
+          <Text mt={2}>
+            {parseContent(decodeHTML(tweet.text)).map((chunk) => {
```

Comment:
should we always parse on every render?

what about moving this to another component?

Reply:
maybe the useMemo can fix the unnecessary rerenders